pyensembl/database.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, gtf, auto_download=False):
     def __eq__(self, other):
         return isinstance(other, Database) and self.gtf == other.gtf
 
+    def __hash__(self):
```

Comment:
Ah ok, it's a file format. Would be good to clear up what the type of`gtf` instance variable is supposed to be though.